tests/integration/ceph_base_mgr_test.go

```diff
@@ -26,17 +28,28 @@ type MgrCLI struct {
 	namespace string
 }
 
+func (cli MgrCLI) checkReadiness() {
+	cmd, args := cephclient.FinalizeCephCommandArgs("ceph", []string{"orch status"}, cli.k8sh.MakeContext().ConfigDir, cli.namespace)
+	output := ""
+	for timeout := 0; timeout < 15; timeout++ {
+		output, err := cli.k8sh.MakeContext().Executor.ExecuteCommandWithOutput(cmd, args...)
+		if err == nil {
+			logger.Info("%+v", output)
+			logger.Info("Rook Toolbox ready to execute commands")
+			break
+		}
+		time.Sleep(1 * time.Second)
+	}
+	logger.Infof("%+v", output)
```

Comment:
I'd suggest logging the output or err during each iteration of the loop so we can troubleshoot the test.

Reply:
ok